azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -486,11 +486,20 @@ private void setupKeyStore() throws ExecutorManagerException {
         throw new RuntimeException("Failed to get hadoop security manager!"
             + e.getCause(), e);
       }
-
-      final KeyStore keyStore = hadoopSecurityManager.getKeyStore(commonPluginLoadProps);
-      if (keyStore == null) {
-        logger.error("Failed to Prefetch KeyStore");
-        throw new ExecutorManagerException("Failed to Prefetch KeyStore");
+      if (commonPluginLoadProps.getBoolean("use.polp.keystores")){
```

Comment:
How about getBoolean("use.polp.keystores", false)? To give a default value there